### PR TITLE
Add support for fifa and randoming teams!

### PR DESCRIPTION
**Pull request by GustavPersson**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -64,13 +67,17 @@ function kartbot(opts) {
           pool = challenge(channel, members, user, args, 'Kart');
           break;
 
+          case (msg.indexOf('!fifa') > -1):
```
Comment (PaulAsjes-zz) [inline index.js]: Indentation

File: package.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
   "name": "kartbot",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "Slack bot for Mario Kart tournaments",
   "license": "MIT",
-  "repository": "PaulAsjes/kartbot",
+  "repository": "GustavPersson/kartbot",
```
Comment (PaulAsjes-zz) [inline package.json]: gusty plz

**Comment by PaulAsjes-zz** [comment]
:+1:

File: package.json (hunk 2 above)
Comment (GustavPersson) [inline package.json]: oops

**Comment by GustavPersson** [comment]
@PaulAsjes Updated!

**Pull request merged**